tools/dockerfile/distribtest/cpp_jessie_x64/Dockerfile

```diff
@@ -27,4 +27,6 @@ RUN apt-get update && apt-get install -y \
       pkg-config \
       unzip && apt-get clean
 
+RUN apt-get update && apt-get install -y cmake golang && apt-get clean
```

Comment:
I'm curious if `golang` from `jessie` is new enough or shall we download from source and install instead?

Reply:
the `golang` here is only to be able to build boringssl using cmake (they are using go to generate some code or config). It's not golang used by gRPC go (and I assume that's what you meant by 1.6 support going away soon).